js/app-super-resolution/screens/mobile.tsx

```diff
@@ -0,0 +1,251 @@
+      return;
+    }
+
+    await imageToPixel(pickerResult.uri)
+
+    return
+  };
+
+
+  async function imageToPixel(uri: string) {
+    const imageResult = await ImageManipulator.manipulateAsync(
+      uri, [
+      { resize: { height: imgHeight, width: imgWidth } }
+    ]
+    )
+
+    const imageScaled = await ImageManipulator.manipulateAsync(
+      uri, [
+      { resize: { height: postImgHeight, width: postImgWidth } }
+    ]
+    )
+
+    bitmapPixel = await bitmapModule.getPixels(imageResult.uri).then(
+
+      (image: any) => {
+        return Array.from(image.pixels);
+      }
+    )
+
+    bitmapScaledPixel = await bitmapModule.getPixels(imageScaled.uri).then(
+      (image: any) => {
+        return Array.from(image.pixels);
+      }
+    )
+
+    setSelectedImage({
+      localUri: imageResult.uri,
+    });
+
+    setOutputImage(null)
+  }
+
+
+  async function openCameraAsync() {
+    const permissionResult = await ImagePicker.requestCameraPermissionsAsync();
+
+    if (permissionResult.granted === false) {
+      alert("Permission to access Camera Roll is Required!");
+      return;
+    }
+
+    const pickerResult = await ImagePicker.launchCameraAsync({ allowsEditing: true });
+
+    if (pickerResult.cancelled === true) {
+      return;
+    }
+
+    await imageToPixel(pickerResult.uri)
+  }
+
+
+  async function preprocess() {
+    floatPixelsY = Float32Array.from(bitmapPixel)
+    cbArray = Array.from(bitmapScaledPixel)
+    crArray = Array.from(bitmapScaledPixel)
+
+    bitmapPixel.forEach((value, index) => {
+      const red = (value >> 16 & 0xFF)
+      const green = (value >> 8 & 0xFF)
+      const blue = (value & 0xFF)
+      floatPixelsY[index] = pixelsRGBToYCbCr(red, green, blue, "y")
+    });
+
+    bitmapScaledPixel.forEach((value, index) => {
+      const red = (value >> 16 & 0xFF)
+      const green = (value >> 8 & 0xFF)
+      const blue = (value & 0xFF)
+      cbArray[index] = pixelsRGBToYCbCr(red, green, blue, "cb")
+      crArray[index] = pixelsRGBToYCbCr(red, green, blue, "cr")
+    })
+
+    let tensor = new ort.Tensor(floatPixelsY, [1, 1, imgHeight, imgWidth])
+    return tensor
+  };
+
+
+  async function postprocess(floatArray: Array<number>): Promise<string> {
+    const intArray = Array<number>(postImgHeight * postImgWidth);
+
+    floatArray.forEach((value, index) => {
+      intArray[index] = (pixelsYCbCrToRGB(value, cbArray[index], crArray[index], platform) as number[])[0]
+    })
+
+    let imageUri = await bitmapModule.getImageUri(intArray).then(
+      (image: any) => {
+        return image.uri
+      }
+    )
+
+    const imageRotated = await ImageManipulator.manipulateAsync(imageUri, [
+      { rotate: 90 },
+      { flip: ImageManipulator.FlipType.Horizontal }
+    ])
+
+    setOutputImage({ localUri: imageRotated.uri })
+    return imageUri
+  };
+
+
+  async function loadModel() {
+    try {
+      const assets = await Asset.loadAsync(require('../assets/super_resnet12.ort'));
```

Comment:
`../assets/super_resnet12.ort` 

How is the model introduced and used in the example? is it loaded from the assets folder?  (um didn't see it in the folder.)

also in tutorial/readme.md file, would be good to mention models used in this example, limitations about it,  other models can try in the future, etc.

Reply:
I just noticed that the model does not get pushed into the repo. So, I added a link to the model in the readme.md file